en/development/dependency-injection.rst

```diff
@@ -280,3 +285,18 @@ stubs::
 Any defined mocks will be replaced in your application's container during
 testing, and automatically injected into your controllers and commands. Mocks
 are cleaned up at the end of each test.
+
+Auto Wiring 
+===============
+
+Auto Wiring is turned off by default. To enable it::
+
+    // In src/Application.php
+    public function services(ContainerInterface $container): void
+    {
+        $container->delegate(
+            new \League\Container\ReflectionContainer()
```

Comment:
Suggesting use of reflection container without caching resolutions (`true` argument) might not be the best idea.
https://container.thephpleague.com/4.x/auto-wiring/

Reply:
Yes this should be handled in a new PR since this one is merged.